### DISCUSSION TITLE
Do we have test cases for latest sunbird version(5.2)? if yes please provide us

### DISCUSSION THREAD

**Discussion by sachinb80** [Sunbird-Ed/Community#484] [General Discussion]
We are migrating to 5.2 latest version from 4.6(the old version).
And need to check what are the new features, upgradation, and fixes in the new or latest version.
[@vinukumar-vs](https://github.com/vinukumar-vs) [@rajeevsathish](https://github.com/rajeevsathish) @[rajeshkumaravel](https://github.com/rajeshkumaravel) [@Krishnaj20](https://github.com/Krishnaj20)

**Comment by chitranshu-keshav** [comment]
@sachinb80 please refer the [release notes](https://ed.sunbird.org/use/releases/release-notes) section of the Sunbird ED microsite for the details of the features/changes in the releases post 4.6. In case you need any additional details then please get back on this thread

**Reply by sachinb80** [reply]
@chitranshu-keshav Yes actually doing that already but I need test cases of the latest version(5.2) also to execute after the migration is done. I need the test cases for testing purposes.  
If u guys have it please provide.

**Reply by chitranshu-keshav** [reply]
@ashokreddy1208 can you help on the above. Please get back to @sachinb80 for any additional information you may need

**Comment by ashokreddy1208** [comment]
@sachinb80 please find 5.2 Regression test cases [here](https://docs.google.com/spreadsheets/d/1W5ywyxnKoyc0jSQO50m5KAuSUnokL9yPxKWE5-8Ug1Y/edit?usp=sharing) 
Refer NF Test Cases sheet for 5.2 new features.

cc @chitranshu-keshav
